config/settings.yml

```diff
@@ -6,4 +6,5 @@ squash_api_host: 'http://squash-host.example.com'
 squash_api_key: 'abc123'
 squash_disable: <%= (Rails.env.development? || Rails.env.test?) %>
 geo_external_url: 'https://earthworks.stanford.edu/catalog/stanford-'
-geo_wms_url: 'https://geowebservices.stanford.edu/geoserver/wms/'
\ No newline at end of file
+geo_wms_url: 'https://geowebservices.stanford.edu/geoserver/wms/'
+was_thumbs_url: 'https://was-thumbnail-prod.stanford.edu/api/v1/was/thumbnails/druid_id'
```

Comment:
Rails_config gem allows you to use a local file for testing `config/settings.local.yml` See https://github.com/railsconfig/rails_config#developer-specific-config-files